### PR TITLE
Package satysfi-base.1.4.0

### PR DESCRIPTION
**Pull request by nyuichi**
### `satysfi-base.1.4.0`
A collection of utility functions and modules for SATySFi
This is a collection of utility functions and modules for SATySFi. Because the library bundled with the default installation configuration of SATySFi is currently not rich enough, this project aims to provide a complementary library sufficient for most situations in typesetting.

this requires Satyrographos to install. See https://github.com/na4zagin3/satyrographos.

---
* Homepage: https://github.com/nyuichi/satysfi-base
* Source repo: git+https://github.com/nyuichi/satysfi-base.git
* Bug tracker: https://github.com/nyuichi/satysfi-base/issues

---
:camel: Pull-request generated by opam-publish v2.0.2
# Automatic follow-ups

- ~~Add to snapshot `snapshot-develop`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)

### CODE REVIEW

**Comment by na4zagin3** [comment]
@nyuichi Do you mind to update the snapshots?

**Comment by nyuichi** [comment]
Sorry, I was a bit busy this weekend. I just updated snapshots following the instructions. Can you check this out again, please?

**Comment by na4zagin3** [comment]
Hmm, something is wrong with the CI. Let me fix it.

File: snapshot-stable-0-0-5.opam [hunk 1]
```diff
@@ -26,7 +26,7 @@ depends: [
 
   "satysfi-azmath" {= "0.0.3"}
 
-  "satysfi-base" {= "1.3.0"}
+  "satysfi-base" {= "1.4.0"}
```
Comment (na4zagin3) [inline snapshot-stable-0-0-5.opam:29]:
```suggestion
  "satysfi-base" {= "1.3.0"}
```

File: snapshot-develop.opam [hunk 2]
```diff
@@ -30,7 +30,7 @@ depends: [
 
   "satysfi-azmath" {= "0.0.3"}
 
-  "satysfi-base" {= "1.3.0"}
+  "satysfi-base" {= "1.4.0"}
```
Comment (na4zagin3) [inline snapshot-develop.opam:33]:
```suggestion
  "satysfi-base" {= "1.3.0"}
```

**Review by na4zagin3 — COMMENTED** [review]
For now we need manually resolve the version conflict.  Let me revert the change.

**Comment by na4zagin3** [comment]
I am sorry but please forgive me reopen this ticket several times to debug the CI scripts.

**Comment by na4zagin3** [comment]
Okay, merging.  Thank you!!

**Comment by na4zagin3** [comment]
Thank you very much for your patience!

**Pull request merged**